docs/spark/tutorials/get-started.md

```diff
@@ -73,14 +67,22 @@ To extract the Apache Spark files:
 
 ![Install Spark](https://dotnet.microsoft.com/static/images/spark-extract-with-7-zip.png?v=YvjUv54LIxI9FbALPC3h8zSQdyMtK2-NKbFOliG-f8M)
 
-Run the following commands to set the environment variables used to locate Apache Spark:
+Run the following commands to set the environment variables used to locate Apache Spark on **Windows**:
 
 ```console
 setx HADOOP_HOME C:\bin\spark-2.4.1-bin-hadoop2.7\
 setx SPARK_HOME C:\bin\spark-2.4.1-bin-hadoop2.7\
 ```
 
-Once you've installed everything and set your environment variables, open a **new** command prompt and run the following command:
+Run the following commands to set the environment variables used to locate Apache Spark on **MacOS** and **Ubuntu**:
+
+```bash
+export SPARK_HOME=~/bin/spark-2.4.1-bin-hadoop2.7/
+export PATH="$SPARK_HOME/bin:$PATH"
+source ~/.bashrc
+```
+
+Once you've installed everything and set your environment variables, open a **new** command prompt or terminal and run the following command:
```

Comment:
I am not sure why I can add suggestions on `### 5. Install .NET for Apache Spark`, but here are my suggestion.
We need to update the worker version to point to .NET Core 3.1
[download the Windows x64 netcoreapp3.1 release](https://github.com/dotnet/spark/releases/download/v0.6.0/Microsoft.Spark.Worker.netcoreapp3.1.win-x64-0.8.0.zip)